src/m365/teams/commands/meeting/meeting-attendacereport-list.spec.ts

```diff
@@ -212,9 +212,7 @@ describe(commands.MEETING_ATTENDANCEREPORT_LIST, () => {
   it('correctly handles error when throwing request', async () => {
     const errorMessage = 'An error has occured';
 
-    sinon.stub(request, 'get').callsFake(async () => {
-      throw { error: { error: { message: errorMessage } } };
-    });
+    sinon.stub(request, 'get').rejects({ error: { error: { message: errorMessage } } });
```

Comment:
Did you see the FileName of this file? If we correct the typo in this PR I wouldn't mind. Ill merge it tomorrow.

Reply:
I did not, but is indeed incorrect. So feel free to correct 😄